charts/auctioneer-test.just

```diff
@@ -0,0 +1,124 @@
+_default:
+  @just --list auctioneer-test
+
+defaultTag := ""
+defaultNamespace := "astria-dev-cluster"
+evm_destination_address := "0x507056D8174aC4fb40Ec51578d18F853dFe000B2"
+smoke_test_image := "bharath123456/auctioneer-smoke-test:latest"
```

Comment:
I pushed the image to my personal dockerhub account for the purpose of a POC. it should be trivial to move it to an ghcr/astria account.